hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/ha/TestHASafeMode.java

```diff
@@ -977,4 +977,31 @@ public void testTransitionToActiveWhenSafeMode() throws Exception {
           () -> miniCluster.transitionToActive(0));
     }
   }
+
+  /**
+   * Test transition to observer when namenode in safemode.
+   *
+   * @throws IOException
```

Comment:
this is inaccurate, it throws `Exception`
but this annotation isn't really necessary IMO? it's just a test